packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -288,6 +288,16 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
         }
     }
 
+    public async newWindow(applicant: string, val: string, windowName: string, windowFeatures: object) {
+        await this.createClient(applicant);
+        const client = this.getBrowserClient(applicant);
+        let args = JSON.stringify(windowFeatures);
```

Comment:
If the documentation is right, the args string should be like this:
`'width=420,height=230,resizable,scrollbars=yes,status=1`
JSON.stringify is not correct one to use.

Reply:
done